src/plugin.cpp

```diff
@@ -169,7 +169,8 @@ int ILLIXR::run(const cxxopts::ParseResult& options) {
        // read in yaml config file
         YAML::Node  config;
         std::string exec_path = get_exec_path();
-        std::string home_dir  = get_home_dir();
+        setenv("ILLIXR_BINARY_PATH", exec_path.c_str(), 1);
```

Comment:
Better use the `cmake_install_prefix/bin` as the binary path. It has trouble finding the vocab file when I ran the executable from my build directory.

Reply:
The CMAKE_INSTALL_PREFIX/bin is not available to C++, unless we add it as a compile definition, which would hard code it. If that is ok, then we can do it that way.

Reply:
I am okay with this. But what do you think would be a better solution?

Reply:
I am not sure.